Makefile

```diff
@@ -69,7 +69,11 @@ EBPF_TESTS='^TestEBPF.*'
 
 .PHONY: go/test
 go/test: $(BIN)/gotestsum
+ifeq ($(GOOS),darwin)
+	$(BIN)/gotestsum -- $(GO_TEST_FLAGS) ./...
+else
 	$(BIN)/gotestsum -- $(GO_TEST_FLAGS) -skip $(EBPF_TESTS) ./... ./ebpf/...
+endif
```

Comment:
This is unrelated but somewhat annoying for Mac users, @korniltsev let me know if you have a more elegant idea here. For context, running `make go/test` on Mac produces this currently:

```
=== Errors
ebpf/python_ebpf_test.go:115:59: undefined: Session
ebpf/python_ebpf_test.go:130:74: undefined: Session
ebpf/python_ebpf_test.go:144:13: undefined: SessionOptions
ebpf/python_ebpf_test.go:161:12: undefined: NewSession
ebpf/python/tss.go:71:17: undefined: mutexSizeGlibc
ebpf/python/tss.go:71:34: undefined: mutexSizeMusl
ebpf/python/versions.go:132:37: undefined: pyVersions
ebpf/python/versions_libc.go:62:17: undefined: glibcOffsets
ebpf/python/versions_libc.go:70:95: undefined: glibcOffsets
ebpf/python/versions_libc.go:71:17: undefined: glibcOffsets
ebpf/python/versions_libc.go:80:29: undefined: glibcOffsets
ebpf/python/versions_libc.go:81:29: undefined: glibcOffsets
ebpf/python/versions_libc.go:98:53: undefined: muslOffsets
ebpf/python/versions_test.go:52:10: undefined: glibcOffsets
ebpf/python/versions_test.go:52:10: too many errors
ld: warning: '/private/var/folders/rl/p7hzvh8943v6xf55qksjjc240000gn/T/go-link-1553094920/000014.o' has malformed LC_DYSYMTAB, expected 98 undefined symbols to start at index 1626, found 95 undefined symbols starting at index 1626
```